recipe/meta.yaml

```diff
@@ -26,8 +26,7 @@ source:
     - patches/0016-Disable-aligned-allocation-on-macOS.patch
     - patches/0017-Drop-strict-python-pkgs-requirements-check.patch
     - patches/0018-Fix-flags-for-un-vendoring-abseil-cpp.patch
-    - patches/0019-wyhash-is-now-called-low_level_hash.patch  # [libabseil != '20210324.2']
-    - patches/0020-Don-t-inject-ICU-Macros-if-using-pre-compiled-librar.patch
+    - patches/0019-Don-t-inject-ICU-Macros-if-using-pre-compiled-librar.patch
```

Comment:
```suggestion
    - patches/0019-Don-t-inject-ICU-Macros-if-using-pre-compiled-librar.patch  # [win]
```

Reply:
but this patch i didn't change.

Reply:
Sorry, then I meant that it should go to 18; in any case, my point was that the patch that switches to `abseil_dll` should only apply to windows.

Reply:
ok, yeah, lets get a working build for windows :D.